src/main/resources/reference.conf

```diff
@@ -16,6 +16,16 @@ jmt {
         probability = 0
       }
     }
+
+    google-cc {
+      min-bw = 30 kbps
+      max-bw = 100 mbps
```

Comment:
The previous default max was 20 mbps, let's keep it unchanged.

Reply:
Done.